cmd/gotk/get_helmrelease.go

```diff
@@ -62,26 +73,52 @@ func getHelmReleaseCmdRun(cmd *cobra.Command, args []string) error {
 		return nil
 	}
 
+	table := tablewriter.NewWriter(os.Stdout)
+	tableRowHeaders := []string{"Name", "Revision", "Ready", "Message", "Suspended"}
+	if allNamespaces {
+		tableRowHeaders = append(tableRowHeaders, "Namespace")
+	}
+	table.SetHeader(tableRowHeaders)
+	table.SetRowLine(true)
```

Comment:
Had a look at the available `tablewriter` configuration options, and the following is what matches the output of `kubectl`:

```go
	table.SetAutoWrapText(false)
	table.SetAutoFormatHeaders(true)
	table.SetHeaderAlignment(tablewriter.ALIGN_LEFT)
	table.SetAlignment(tablewriter.ALIGN_LEFT)
	table.SetCenterSeparator("")
	table.SetColumnSeparator("")
	table.SetRowSeparator("")
	table.SetHeaderLine(false)
	table.SetBorder(false)
	table.SetTablePadding("\t")
	table.SetNoWhiteSpace(true)
```

This renders:

```console
$ ./bin/gotk get helmreleases -n helm-system
NAME    REVISION        READY   MESSAGE                                                 SUSPENDED 
podinfo                 False   Helm test failed: timed out waiting for the condition   False
```

---

You probably want to factor this out to `utils.go` to prevent repetitive code. E.g:

- Prepare just the configuration: `func (*Utils) newTableWriter(writer io.Writer) *tablewriter.Table`)
- Do everything in one run (likely the better option): `func (*Utils) printTable(writer io.Writer, header []string, rows [][]string)`. The contents of the function would look something like the following (pseudocode):

  ```go
    table := tablewriter.NewWriter(writer)
	table.SetHeader(header)
	// configure options from above
	table.AppendBulk(rows)
	table.Render()
  ```

  In the commands, the code would then become something like this (pseudocode):

  ```go
  header := []string{"Name", ...}
  var rows [][]string
  for _, i := range list.Items {
	row := []string{
    	i.GetName(), ...
	}
    rows = append(rows, row)
  }
  utils.printTable(io.Stdout, header, rows)
  ```